src/js/tabs/receiver.js

```diff
@@ -371,6 +371,10 @@ receiver.initialize = function (callback) {
                 }
                 updateSaveButton(true);
             });
+
+            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_47)) {
+                $('input[name="elrsModelId-number"]').val(FC.RX_CONFIG.elrsModelId);
+            }
```

Comment:
else hide input field.

Reply:
will it retain hidden state for new FC? i forget/didnt' test.

Reply:
Sure it wil reload / evaluate the page when switching FC.